src/compose.ts

```diff
@@ -493,7 +493,6 @@ export interface ComposeFactory<T, O extends Options> extends ComposeMixinable<T
 	/**
 	 * Extend the factory prototype with the supplied object literal, class, or factory
 	 *
-	 * @deprecated
```

Comment:
Quick question on one of the examples, they all made sense and assuming backwards compatibility I think that I'm happy with the approach.

```ts
const createFoo = compose({
	foo: 'original value'
}, function(instance, options?: { foo: string }) {
	instance.foo = 'initialized value';
});

const bar = compose.createMixin(createFoo)
	.extend({
		bar: 'bar'
	})
	.init((instance, options?: { foo: string, bar: string ) => {
		if (options) {
			instance.bar = options.bar;
			instance.foo = options.foo;
		}
	});

const createFooBar = createFoo.mixin(bar);

const fooBar = createFooBar({
	foo: 'final value',
	bar: 'new value'
});
```

Wouldn't `bar` be the same as `createFooBar`?